wdl-format/src/token/post.rs

```diff
@@ -44,17 +46,18 @@ impl std::fmt::Display for PostToken {
         match self {
             PostToken::Space => write!(f, "{SPACE}"),
             PostToken::Newline => write!(f, "{NEWLINE}"),
+            PostToken::Indent => write!(f, "    "), // 4 spaces TODO replace
```

Comment:
```suggestion
            PostToken::Indent => write!(f, "    "), // TODO(af): replace me
```

Reply:
I usually search "TODO:" or "TODO(clay):", so it might be good to just get in the habit of that.